site/en/blog/new-in-extensions-2/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: 'layouts/blog-post.njk'
+title: An example blog post
+description: >
+  Open source tool to convert extensions to Manifest V3. You’ll still need to manually update any service worker code that relies on a DOM, in addition to updating chrome.scripts.
```

Comment:
I don't know exactly where we landed on this—I have strong opinions—but right now we don't allow smart quotes (’). Please replace with ' or ".

Reply:
SG